web_src/js/features/user-auth-webauthn.ts

```diff
@@ -174,6 +174,10 @@ async function webauthnRegistered(newCredential) {
 function webAuthnError(errorType, message) {
   const elErrorMsg = document.querySelector(`#webauthn-error-msg`);
 
+  if (!elErrorMsg) {
```

Comment:
looks unrelated could you move it into another pull?

Reply:
just a small nit, looks not necessary to create a new pull request.